docs/guides/runtime-options-overview.mdx

```diff
@@ -206,29 +205,13 @@ For more information about the primitive options, refer to the
 
 The Qiskit Runtime primitives expect to be called with circuits already suitable for execution on the target system. This implies that the user has already transpiled their circuits to respect the native gate set and connectivity constraints of the target system.
 
-The Qiskit Runtime primitives may perform additional runtime compilation to optimize circuits, with the degree of optimization controlled by an optimization level option. The optimization level you choose affects the compilation strategy, with higher levels invoking more expensive or aggressive optimizations.
+V2 primitives do not support any advanced transpilation options or optimization levels. For more information and a complete list of advanced transpilation options, see the Advanced transpilation options table in the
```

Comment:
This paragraph sounds weird to me. It says V2 doesn't support advanced transpilation options, yet it points me to a complete list of advanced transpilation options 🤔